### PR TITLE
GAWB-2488: Update owners when changing method/config permissions

### PR DESCRIPTION
**Pull request by MattPutnam**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [x] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/method_repo/method/common.cljs [hunk 1]
```diff
@@ -78,14 +78,14 @@
...
-    (style/create-summary-block (str "Config Owner" (when (> (count managers) 1) "s"))
+    (style/create-summary-block (str "Configuration Owner" (when (> (count managers) 1) "s"))
                                 (string/join ", " managers))
     (style/create-summary-block "Designed For" (str "Method Snapshot " (:snapshotId method)))]
    [:div {:style {:display "flex"}}
     (style/create-summary-block "Root Entity Type" (:rootEntityType payloadObject))
     ;; Snapshot comments for configs can only be created by the API. Hide the comment field if it doesn't
     ;; exist to avoid tantalizing UI-only users with something they can't have (at least until GAWB-2702)
-    (if snapshotComment (style/create-summary-block "Snapshot Comment" snapshotComment))]
+    (some->> snapshotComment (style/create-summary-block "Snapshot Comment"))]
```
Comment (aednichols) [inline src/cljs/main/broadfcui/page/method_repo/method/common.cljs:88]: Thanks

**Comment by davidangb** [comment]
In testing, when changing permissions on a config, I get a js error on save. The permissions appear to save correctly, but I still get the error:
```
Uncaught TypeError: Cannot read property 'call' of null
    at configs.cljs?rel=1507658797761:72
    at Function.<anonymous> (methods_configs_acl.cljs?rel=1507658797523:140)
    at Function.cljs.core.apply.cljs$core$IFn$_invoke$arity$3 (core.cljs:3727)
    at cljs$core$apply (core.cljs:3709)
    at Function.dmohs.react.core.call_fn.cljs$core$IFn$_invoke$arity$variadic (core.cljs?rel=1507224342331:218)
    at dmohs$react$core$call_fn (core.cljs?rel=1507224342331:217)
    at Object.G__28368__delegate (core.cljs?rel=1507224342331:233)
    at Object.G__28368 [as -post-update] (core.cljs?rel=1507224342331:233)
    at Function.dmohs.react.core.call.cljs$core$IFn$_invoke$arity$variadic (core.cljs?rel=1507224342331:140)
    at Function.dmohs.react.core.call.cljs$lang$applyTo (core.cljs?rel=1507224342331:136)
```

**Review by aednichols — APPROVED** [review]
Tested & works 👍

**Comment by MattPutnam** [comment]
@davidangb -- refresh harder?

**Review by davidangb — APPROVED** [review]
👍 wfm!

**Pull request merged**